PyInstaller/compat.py

```diff
@@ -437,3 +437,11 @@ def importlib_load_source(name, pathname):                # Import module from a
         mod_loader = importlib.machinery.SourceFileLoader(name, pathname)
         return mod_loader.load_module()
 
+# String types to replace `isinstance(foo, str)`
+# Use `isinstance(foo, string_types)` instead.
+
+if is_py2:
```

Comment:
I'd prefer to *not* have such a thing here for two reasons:
a) in Python 2, there is `basestring` for this
b) it's only used in `winmanifest.py`